devtools/gradle/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Sat Mar 21 13:51:47 BRT 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.2-all.zip
```

Comment:
This is necessary to navigate through the sources while debugging the plugin in IntelliJ